runtime/moonbase/tests/common/mod.rs

```diff
@@ -260,3 +262,32 @@ pub fn set_parachain_inherent_data() {
 	)
 	.dispatch(inherent_origin()));
 }
+
+pub fn uxt(valid: bool) -> UncheckedExtrinsic {
```

Comment:
Name too short and making the test unreadable. It is better to rename this function `unchecked_eth_tx` and have it take `raw_hex_tx: &str`